code/game/area/Space Station 13 areas.dm

```diff
@@ -28,7 +28,7 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	var/lightswitch = 1
 
 	var/eject = null
-
+	var/is_maintenance = FALSE
```

Comment:
this new var simplifies a lot of maintenance checking logic, and allows any area to be designated maint